appserver/web/weld-integration/src/main/java/org/glassfish/weld/JaxRSJsonContextResolver.java

```diff
@@ -81,8 +81,8 @@ private JaxRSJsonContextResolver(List<ContextResolver<?>> existingResolvers) {
     @Override
     public void configure(FeatureContext context) {
         List<ContextResolver<?>> resolvers = context.getConfiguration().getInstances().stream()
-                .filter(o -> ContextResolver.class.isAssignableFrom(o.getClass()))
-                .map(o -> (ContextResolver<?>) o)
+                .filter(ContextResolver.class::isInstance)
+                .map(resolver -> (ContextResolver<?>) resolver)
```

Comment:
You could also avoid this lambda by using ContextResolver.class::cast

Reply:
Same thing I told Markus, due to generics, this will not work

Reply:
ok